### PR TITLE
Mono and Python: Always use latest certs from trust store. Fix LE cert

### PR DESCRIPTION
**Pull request by publicarray**
Python: https://pypi.org/project/certifi/

Fixes: https://discord.com/channels/732558169863225384/732559466721181738/894569903737163836

### CODE REVIEW

File: spk/mono/src/service-setup.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 # mono service setup
 
-service_postinst ()
+service_prestart ()
 {
     # Sync ca certificates
     ${SYNOPKG_PKGDEST}/bin/cert-sync /etc/ssl/certs/ca-certificates.crt
```
Comment (publicarray) [inline spk/mono/src/service-setup.sh:7]: I still need to find out if this works with DSM7 as I'm not 100% sure about the permissions

Comment (publicarray) [reply spk/mono/src/service-setup.sh:7]: A new release would also fix this, but I think this is better in the long run.

Comment (publicarray) [reply spk/mono/src/service-setup.sh]: What was I thinking. mono isn't a daemon.

**Comment by hgy59** [comment]
@publicarray I unsure whether is is worth to remove the explicit version from certifi dependency.

I still prefer to nail those to an explicit version with ==. As we cannot update a package without incrementing the SPK_REV we also can use a specific version of certify in the packages (obviously the current version at publishing date).
Without this, it is not possible to reproduce a specific spk version.

**Comment by hgy59** [comment]
and it would be better not to update packages that are under development (octoprint, homeassistant, ...) to avoid merge conflicts or packages that fail to build on the master branch.
As there is a redesign ongoing to separate pure python and cross compiled wheels (@th0ma7), a lot of merge conflicts are predictable.

**Comment by publicarray** [comment]
Yea fair point.

**Comment by publicarray** [comment]
@hgy59 For consistency these should have been set. I don't mind which way we co, but I do prefer consistency.

https://github.com/SynoCommunity/spksrc/blob/9e02f76eb6bd5afffbead8b3903fa307ee174bd1/spk/fishnet/src/requirements.txt#L9

https://github.com/SynoCommunity/spksrc/blob/9e02f76eb6bd5afffbead8b3903fa307ee174bd1/spk/gateone/src/requirements.txt#L1-L6

https://github.com/SynoCommunity/spksrc/blob/9e02f76eb6bd5afffbead8b3903fa307ee174bd1/spk/python2/src/requirements.txt#L31

**Pull request closed without merging**